tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs

```diff
@@ -0,0 +1,480 @@
+            self.full_propagate_parallelized(numerator)
+        }
+
+        self.unchecked_scalar_rem_parallelized(numerator, divisor)
+    }
+
+    pub fn smart_scalar_div_rem_parallelized<T>(
+        &self,
+        numerator: &mut RadixCiphertext,
+        divisor: T,
+    ) -> (RadixCiphertext, RadixCiphertext)
+    where
+        u64: From<T>,
+        T: UnsignedInteger + DecomposableInto<u8>,
+    {
+        if !numerator.block_carries_are_empty() {
+            self.full_propagate_parallelized(numerator)
+        }
+
+        self.unchecked_scalar_div_rem_parallelized(numerator, divisor)
+    }
+
+    pub fn scalar_div_assign_parallelized<T>(&self, numerator: &mut RadixCiphertext, divisor: T)
+    where
+        u64: From<T>,
+        T: UnsignedInteger + DecomposableInto<u8>,
+    {
+        if !numerator.block_carries_are_empty() {
+            self.full_propagate_parallelized(numerator)
+        }
+
+        *numerator = self.unchecked_scalar_div_parallelized(numerator, divisor);
+    }
+
+    pub fn scalar_rem_assign_parallelized<T>(&self, numerator: &mut RadixCiphertext, divisor: T)
+    where
+        u64: From<T>,
+        T: UnsignedInteger + DecomposableInto<u8>,
+    {
+        if !numerator.block_carries_are_empty() {
+            self.full_propagate_parallelized(numerator)
+        }
+
+        *numerator = self.unchecked_scalar_rem_parallelized(numerator, divisor);
+    }
+
+    /// Computes homomorphically a division of a ciphertext by a scalar
+    ///
+    /// # Note
+    ///
+    /// If you need both the quotien and the remainder of the division
+    /// use [Self::scalar_div_rem_parallelized]
+    ///
+    /// # Panics
+    ///
+    /// panics if scalar is zero
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use tfhe::integer::gen_keys_radix;
+    /// use tfhe::shortint::parameters::PARAM_MESSAGE_2_CARRY_2;
+    ///
+    /// // We have 4 * 2 = 8 bits of message
+    /// let size = 4;
+    /// let (cks, sks) = gen_keys_radix(PARAM_MESSAGE_2_CARRY_2, size);
+    ///
+    /// let msg = 230u8;
+    /// let scalar = 12u8;
+    ///
+    /// let ct = cks.encrypt(msg);
+    ///
+    /// // Compute homomorphically a scalar division:
+    /// let ct_res = sks.scalar_div_parallelized(&ct, scalar);
+    ///
+    /// let decrypted: u8 = cks.decrypt(&ct_res);
+    /// assert_eq!(msg / scalar, decrypted);
+    /// ```
+    pub fn scalar_div_parallelized<T>(
+        &self,
+        numerator: &RadixCiphertext,
+        divisor: T,
+    ) -> RadixCiphertext
+    where
+        u64: From<T>,
+        T: UnsignedInteger + DecomposableInto<u8>,
+    {
+        let mut result = numerator.clone();
+        self.scalar_div_assign_parallelized(&mut result, divisor);
+        result
+    }
+
+    /// Computes homomorphically the remainder of the division of a ciphertext by a scalar
+    ///
+    /// # Note
+    ///
+    /// If you need both the quotien and the remainder of the division
```

Comment:
quotien -> quotient, maybe add `.` at the end of the docstrings sentences ? (valid for all dcstrings in the file).